webapp2.py

```diff
@@ -119,7 +123,7 @@
     505: 'HTTP Version not supported',
 }
 status_reasons.update(_webapp_status_reasons)
-for code, message in _webapp_status_reasons.iteritems():
+for code, message in _webapp_status_reasons.items():
```

Comment:
`.items()` isn't an iterator in Python 2, so if `_webapp_status_reasons` is super huge, then we load them all into memory. However, it looks like the list isn't that large (7 items).